website/README.md

```diff
@@ -30,164 +29,39 @@ Your project file structure should look something like this
 ```
 my-docusaurus/
   docs/
-    doc-1.md
-    doc-2.md
-    doc-3.md
+    cli.md
   website/
     blog/
-      2016-3-11-oldest-post.md
-      2017-10-24-newest-post.md
     core/
     node_modules/
     pages/
     static/
       css/
       img/
+    versioned_docs/
+    versioned_sidebars/
+    README.md
     package.json
     sidebar.json
     siteConfig.js
+    versions.json
```

Comment:
There's a bash utility called `tree` that helps to make beautiful project structures. You can install it using `brew install tree` and then generate the file structure. This is just being picky but I think is a nice addition to the docs

Reply:
Done!!!